cmd/integration_test/build/buildspecs/quick-test-eks-a-cli.yml

```diff
@@ -42,11 +45,39 @@ env:
     T_VSPHERE_CIDR: "vsphere_ci_beta_connection:vsphere_cidr"
     T_VSPHERE_PRIVATE_NETWORK_CIDR: "vsphere_ci_beta_connection:vsphere_private_network_cidr"
     T_VSPHERE_RESOURCE_POOL: "vsphere_ci_beta_connection:resource_pool"
-    T_VSPHERE_SERVER: "vsphere_ci_beta_connection:server"
```

Comment:
Have you verified this variable is not needed?

Reply:
Yes, this was a duplicate in fact.